inc/styles-admin.php

```diff
@@ -228,7 +228,7 @@ class="measurement-unit measurement-unit-<?php echo ! empty( $field['multiple']
 					$image = SiteOrigin_Panels_Styles::get_attachment_image_src( $current, 'thumbnail' );
 				}
 				
-				$fallback_url = $current_styles[ $field_id . '_fallback' ];
+				$fallback_url = ( ! empty( $current_styles[ $field_id . '_fallback' ] ) ? $current_styles[ $field_id . '_fallback' ] : '' );
```

Comment:
The field can contain the string 'false' as it's value, hence the check `$fallback_url !== 'false'` below. Please add that here.

Reply:
It's okay if it's false as empty will account for that.

> The following values are considered to be empty:
> [...]
> FALSE

(from https://secure.php.net/manual/en/function.empty.php)

Reply:
Please see the first [type comparison table](https://secure.php.net/manual/en/types.comparisons.php) **Comparisons of $x with PHP functions** The last entry shows `empty('false')` evaluates to FALSE.

Reply:
Why would it ever return false as a string? 😕